frontend/app/routes/$lang+/_protected+/personal-information+/index.tsx

```diff
@@ -42,6 +44,7 @@ export async function loader({ context: { session }, params, request }: LoaderFu
 
   const raoidcService = await getRaoidcService();
   await raoidcService.handleSessionValidation(request, session);
+  const locale = getLocale(request);
```

Comment:
```suggestion
  const locale = await getLocale(request);
```

(and remove `await` from the code below)

Reply:
Fixed: c643167653705749ea9866676ea2487dc7f57182

Reply:
Yeah, that was accidental added. Pushed: c643167653705749ea9866676ea2487dc7f57182

Reply:
Yeah, that was accidental added. Pushed: https://github.com/DTS-STN/canadian-dental-care-plan/commit/c643167653705749ea9866676ea2487dc7f57182